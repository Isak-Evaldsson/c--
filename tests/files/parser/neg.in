int test() {
    int a = -1;
    int b = -8 * 2;
    int c = -(8 * 2);   
}