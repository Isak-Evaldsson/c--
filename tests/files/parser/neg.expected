Prototype: test
    VarDecl: a
        NegationExpr: 
            LiteralExpr: 1
    VarDecl: b
        BinOpExpr: *
            NegationExpr: 
                LiteralExpr: 8
            LiteralExpr: 2
    VarDecl: c
        NegationExpr: 
            BinOpExpr: *
                LiteralExpr: 8
                LiteralExpr: 2
